.gitignore

```diff
@@ -11,10 +11,13 @@ project/plugins/project/
 .history
 .cache
 .lib/
+.idea/
```

Comment:
Build related files should not be put into a projects `.gitignore`, instead they should be done on a local level using a global gitignore (see https://stackoverflow.com/a/7335487)

Reply:
I wanted to at one point actually go through all of the projects and remove them and document to use a global git ignore

Reply:
I removed the .gitignore changes